core/templates/components/certificate-assessment-offering-helper/certificate-offering-review-and-availability.component.css

```
@@ -0,0 +1,206 @@
+  font-size: 14px;
+  font-weight: 600;
+  padding: 8px 20px;
+}
+
+.oppia-certificate-review-btn--save:hover {
+  background: #004d47;
+}
+
+.oppia-visually-hidden {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)